src/main/webapp/WEB-INF/datastore-indexes.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<datastore-indexes autoGenerate="false">
```

Comment:
The thing is that they will be added to `appengine-generated/datastore-indexes-auto.xml`, which isn't source controlled. Setting to false will cause the dev server to fail, exactly like the live server. If someone is adding a new feature that requires a new index, they will have to update `datastore-indexes.xml`, which is source controlled. They can still do so by temporarily setting to true then copying the relevant ​lines from `appengine-generated/datastore-indexes-auto.xml` over.